data_steward/analytics/cdr_ops/raw_rdr_export_qc.py

```diff
@@ -1297,4 +1297,18 @@
             new_rdr=new_rdr)
 execute(client, query)
 
+# # Check to catch duplicate observation_ids
+
+tpl = JINJA_ENV.from_string('''
+SELECT
+  observation_id,
+  COUNT(DISTINCT observation_id) AS n
```

Comment:
You can drop the distinct in the count too, sorry should've caught this earlier.